modules/os-datagrid-online-services-configuration/src/main/bash/jdg-online-configuration.sh

```diff
@@ -18,41 +18,60 @@ function check_if_profile_exists() {
 }
 
 function check_if_cli_exists() {
-   if [ ! -f "${CLI}" ]; then
-      echo "CLI $CLI doesn't exist."
+   if [ ! -f "${JBOSS_HOME}/bin/cli.sh" ]; then
+      echo "CLI ${JBOSS_HOME}/bin/cli.sh doesn't exist."
       exit 1
    fi
 }
 
-while [ "$1" != "" ]; do
-    PARAM=`echo $1 | awk -F= '{print $1}'`
-    VALUE=`echo $1 | awk -F= '{print $2}'`
-    case $PARAM in
-        -h | --help)
-            usage
-            exit
-            ;;
-        --profile)
-            PROFILE=$VALUE
-            ;;
-        --profiles-directory)
-            PROFILES_DIRECTORY=$VALUE
-            ;;
-        --cli-bin)
-            CLI=$VALUE
-            ;;
-        *)
-            echo "ERROR: unknown parameter \"$PARAM\""
-            usage
-            exit 1
-            ;;
-    esac
-    shift
+function flush_additional_properties_to_file() {
+   echo "" > "$JBOSS_HOME/bin/.jbossclirc"
```

Comment:
`touch` ?

Reply:
No, I actually want to remove any content that is already there.